corehq/messaging/management/commands/import_conditional_alerts.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from corehq.apps.data_interfaces.models import (
+    AutomaticUpdateRule,
+    MatchPropertyDefinition,
+    CreateScheduleInstanceActionDefinition,
+)
+from corehq.apps.domain.models import Domain
+from corehq.messaging.management.commands.export_conditional_alerts import (
+    SimpleSchedulingRule,
+    SimpleSMSDailyScheduleWithTime,
+    SimpleSMSAlertSchedule,
+    SIMPLE_SMS_DAILY_SCHEDULE_WITH_TIME,
+    SIMPLE_SMS_ALERT_SCHEDULE,
+)
+from corehq.messaging.scheduling.models import (
+    AlertSchedule,
+    TimedSchedule,
+    TimedEvent,
+    SMSContent,
+)
+from corehq.messaging.tasks import initiate_messaging_rule_run
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+import json
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('filename')
+
+    def handle(self, domain, filename, **options):
+        domain_obj = Domain.get_by_name(domain)
+        if domain_obj is None:
+            raise CommandError("Project space '%s' not found" % domain)
+
+        if not domain_obj.uses_new_reminders:
+            raise CommandError("Project space '%s' does not have new reminders enabled" % domain)
+
+        json_rules = []
+        with open(filename, 'rb') as f:
```

Comment:
Should say `with open(path, encoding="utf-8") as f:` [to support both Python 2 and 3](https://github.com/dimagi/langcodes/pull/19#discussion_r189239788)

Reply:
updated for py3 compatibility here: https://github.com/dimagi/commcare-hq/pull/21366/commits/606b606f75b6e19af4dc24038fece9713af6318d